pkg/registry/registry_test.go

```diff
@@ -0,0 +1,131 @@
+package registry
```

Comment:
are these tests ran in ci yet?

Reply:
yes, all tests in the server are run by ci